### DISCUSSION TITLE
CIDR change to reference architecture

### DISCUSSION THREAD

**Discussion by velarx** [gruntwork-io/knowledge-base#582] [Help] [answered]
Currently we had got setup the reference architecture with the default CIDR for the dev, stage and prod and we want to change those CIDR to reserved CIDR. Is that could impact to whole architecture. Also, I want to know if we are following per account per application structure in that case what CIDR range would be required for the dev, stage and prod instead of /16 range.

---

<ins datetime="2022-11-03T10:00:40Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109521">Tracked in ticket #109521</a></p>
</ins>

**Comment by josh-padnick** [comment, marked as answer]
We answered a similar question in https://github.com/gruntwork-io/knowledge-base/discussions/600.
